### PR TITLE
Add onelight theme variant

### PR DESCRIPTION
**Pull request by erasin**

### CODE REVIEW

File: runtime/themes/onelight.toml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+"ui.cursor.insert" = { bg = "white" }
+
+"ui.highlight" = { bg = "light-white" }
+
+"ui.selection" = { bg = "light-white" }
+"ui.selection.primary" = { bg = "light-white", modifiers = ["dim"] }
+
+"ui.virtual" = { fg = "light-white"}
+# "ui.virtual.ruler" = {}
+# "ui.virtual.whitespace" = {}
```
Comment (the-mikedavis) [inline runtime/themes/onelight.toml:86]: For these it's typical to highlight `ui.virtual.whitespace` as `{ fg = "light-white" }` but have `ui.virtual.ruler` as `{ bg = "light-white" }`. It's up to you though, a foreground ruler can also make sense if you don't want to see the ruler until characters are on the ruler. What do you think?

(FYI: for testing these, if you're building from latest you can do `:set rulers [80]` to turn on a ruler on column 80 and `:set whitespace.render all` for the visible whitespace).

Comment (erasin) [reply runtime/themes/onelight.toml]: Do not hide virtual characters, but do not hinder the code when use editor.   I haven't used rulers yet, so ...
I change theme file ,add `"ui.virtual.ruler" = { bg = "light-white" }`.

**Review by the-mikedavis — APPROVED** [review]
looks good, thanks!

**Comment by the-mikedavis** [comment]
Don't forget to add a screenshot to the themes wiki: https://github.com/helix-editor/helix/wiki/Themes

**Pull request merged**